eng/pipelines/templates/steps/analyze.yml

```diff
@@ -13,11 +13,16 @@ steps:
      ward scan -d $(Build.SourcesDirectory) -c $(Build.SourcesDirectory)/eng/.docsettings.yml
     displayName: 'Verify Readmes'
 
+  - pwsh: |
+      mkdir "$(Build.ArtifactStagingDirectory)/reports"
+      Copy-Item -Path "$(Build.SourcesDirectory)/eng/common/InterdependencyGraph.html" -Destination "$(Build.ArtifactStagingDirectory)/reports/InterdependencyGraph.html"
+    displayName: 'Create Reports Staging Folder'
```

Comment:
Nit: probably should rename to `Populate`, given that this graph is immediately getting copied into it.